packages/eas-cli/src/submissions/android/AndroidSubmissionConfig.ts

```diff
@@ -0,0 +1,33 @@
+export interface AndroidSubmissionConfig {
+  projectId: string;
+  archiveUrl: string;
+  archiveType: ArchiveType;
+  androidPackage: string;
+  track: ReleaseTrack;
+  serviceAccount: string;
+  releaseStatus?: ReleaseStatus;
+}
+
+export enum AndroidArchiveType {
+  apk = 'apk',
+  aab = 'aab',
+}
+export enum IosArchiveType {
+  ipa = 'ipa',
+}
+
+export type ArchiveType = AndroidArchiveType | IosArchiveType;
```

Comment:
This shouldn't be inside `android` directory.

Reply:
Also, this seems irrelevant to moving the Android part of `upload:*`. I feel it shouldn't be included in this PR.